codegen/src/test/resources/software/amazon/awssdk/codegen/poet/client/c2j/json-query-error/customization.config

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Can we rename this folder to
query-to-json-errorcode

Reply:
done